Library/Homebrew/cmd/migrate-linux-core.rb

```diff
@@ -0,0 +1,48 @@
+# typed: false
+# frozen_string_literal: true
+
+require "utils/popen"
+
+module Homebrew
+  extend T::Sig
+
+  MIGRATION_CACHE_FILE = (HOMEBREW_CACHE/"linuxbrew-core-migration-done").freeze
```

Comment:
This file might disappear after `brew cleanup` I think

Reply:
Maybe better use git config for that

Reply:
> Maybe better use git config for that

Agreed. You don't even need any new files/configuration for that, you can just use the Git remote URL.

Reply:
I need to keep the state of the migration somewhere. If the migration failed or canceled, I need to run it again. The remote url might have been changed but the formulae might not all have been migrated. So I need a file somewhere to keep track of the migration status.

Reply:
Is there a better place to keep a cache file around that will never be deleted?